src/xc/xc_libxc_wrap.F

```diff
@@ -266,31 +266,31 @@ INTEGER FUNCTION xc_libxc_wrap_functional_get_number(func_string) RESULT(func_id
    END FUNCTION xc_libxc_wrap_functional_get_number
 
 ! **************************************************************************************************
-!> \brief Wrapper to test wether functional is considered stable in Libxc
+!> \brief Wrapper to test wether functional is considered under development in Libxc
 !> \param xc_info ...
 !>
 !> \return ...
 !> \author F. Stein (fstein93)
 ! **************************************************************************************************
-   LOGICAL FUNCTION xc_libxc_wrap_is_stable(xc_info)
+   LOGICAL FUNCTION xc_libxc_wrap_is_under_development(xc_info)
       TYPE(xc_f03_func_info_t)                           :: xc_info
 
-      CHARACTER(LEN=*), PARAMETER :: routineN = 'xc_libxc_wrap_is_stable', &
+      CHARACTER(LEN=*), PARAMETER :: routineN = 'xc_libxc_wrap_is_under_development', &
          routineP = moduleN//':'//routineN
 
       INTEGER                                            :: handle
 
       CALL timeset(routineN, handle)
```

Comment:
please remove `timeset` and `timestop` here in a second commit while being at it, there is no point in measuring time for this part